ts/theme/variables.ts

```diff
@@ -142,11 +149,22 @@ const customVariables = Object.assign(materialVariables, {
   textBoldWeight: "700" as FontWeight,
   textLinkWeight: "600" as FontWeight,
 
+  // Item
+  itemPaddingTopSpacer: 20,
+
+  // Message
+  iconSingleMessagePaddingTop: 10,
```

Comment:
instead of defining variables in the theme that are specific for a single screen (i.e. `Message`) can we have the screen derive these values from other shared theme variables? - cc @fpersico